src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -488,7 +488,7 @@ private void validateBranches(boolean minImpedance, boolean dc) {
                 LfBus bus1 = branch.getBus1();
                 LfBus bus2 = branch.getBus2();
                 // ensure target voltages are consistent
-                if (bus1 != null && bus2 != null) {
+                if (bus1 != null && bus2 != null && bus1.isVoltageControlled() && bus2.isVoltageControlled()) {
```

Comment:
@annetill to handle in a separate PR, but this check is buggy and has to be fixed. Also I am not sure it is needed because we already check this at network loading.

Reply:
But it does not check it is regulating